rc/extra/gdb.kak

```diff
@@ -174,7 +174,7 @@ define-command gdb-session-stop %{
                 rm "${kak_opt_gdb_dir}/pid" "${kak_opt_gdb_dir}/pipe"
                 rmdir "$kak_opt_gdb_dir"
             else
-                printf %s\\n raise
+                echo raise
```

Comment:
Okay, I did it this way because in general I prefer writing kakoune commands not in a shell scope if I can avoid it, but I don't mind changing it.